sdk/internal/recording/server.go

```diff
@@ -0,0 +1,329 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package recording
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"runtime"
+	"time"
+)
+
+func getTestProxyDownloadFile() (string, error) {
+	if runtime.GOOS == "windows" {
```

Comment:
Why is windows excluded from the switch statements below? Not good enough?

Reply:
We don't build an ARM binary for windows, so we can just return x64 right away.

Reply:
You don't _have_ to compare the arch in your case statement?

Reply:
you're so right, I think originally it was written a bit differently with a helper which is why I had it this way. Updated to just use the switch.